snuba/cli/devserver.py

```diff
@@ -74,6 +74,33 @@ def devserver(*, bootstrap: bool, workers: bool) -> None:
                 "--storage=events",
             ],
         ),
+        (
+            "subscriptions-consumer-events",
+            [
+                "snuba",
+                "subscriptions",
+                "--auto-offset-reset=latest",
+                "--log-level=debug",
+                "--max-batch-size=1",
+                "--topic=events-subscription-results",  # TODO: Figure out what this should be
```

Comment:
`topic` is `events` for both as there is only one topic to consume events and transactions today.